test/interactions/dragInteractionTests.ts

```diff
@@ -156,6 +145,49 @@ describe("Interactions", () => {
       assert.deepEqual(receivedStart, startPoint, "was passed the correct starting point");
       assert.deepEqual(receivedEnd, endPoint, "was passed the correct current point");
 
+      assert.strictEqual(drag.onDragEnd(), endCallback, "retrieves the callback if called with no arguments");
+      drag.onDragEnd(null);
+      assert.isNull(drag.onDragEnd(), "removes the callback if called with null");
+      svg.remove();
+    });
+
+    it("constrain()", () => {
```

Comment:
I could make this a `describe()` and have sub-`describe()`s for the touch and mouse cases.